forge-gui/res/cardsfolder/b/backslide.txt

```diff
@@ -1,7 +1,7 @@
 Name:Backslide
 ManaCost:1 U
 Types:Instant
-A:SP$ SetState | Cost$ 1 U | ValidTgts$ Creature.faceUp+withMorph,Creature.faceUp+withMegamorph | TgtPrompt$ Select target creature with morph. | Mode$ TurnFace | SpellDescription$ Turn target creature with a morph ability face down.
+A:SP$ SetState | Cost$ 1 U | ValidTgts$ Creature.withMorph,Creature.withMegamorph | TgtPrompt$ Select target creature with morph. | Mode$ TurnFaceDown | SpellDescription$ Turn target creature with a morph ability face down.
```

Comment:
rules wise this still should only be target face up cards?

Reply:
they only say "with morph ability" so face-down copy of one should be allowed imo (for all the good that it would do)

Reply:
my question is more of a rules thing, like does a facedown card count as "with morph ability" ?

Reply:
I could not find any information to indicate that it should let you target a normal face-down morph, so no further changes for now

Reply:
does it mean you keep it as `Creature.faceUp+withMorph` for now?

Reply:
I think I should still remove it because the card doesn't say so
this does not change any behaviour ingame

Reply:
its the difference if the spell is allowed to target a face down card or not

Reply:
"with a morph ability" just means "with morph" (702.1d)
so just checking for KW seems right
and if the facedown could somehow gain it, it should be allowed

Reply:
hm it might be from a time when we had morph copied to face down state.

have you tried to target a face down card with this?

Reply:
- it still can't target face-down morph
- when it targets face-up morph it can be unmorphed fine